package.json

```diff
@@ -98,6 +99,7 @@
     "cypress": "4.5.0",
     "cypress-file-upload": "^4.1.1",
     "eslint-plugin-cypress": "^2.11.3",
-    "start-server-and-test": "^1.12.1"
+    "start-server-and-test": "^1.12.1",
+    "tailwindcss": "^2.2.4"
```

Comment:
We don't use Tailwind CSS in this project to style the components.

Reply:
ok, I will remove it.